ds701_book/11-Dimensionality-Reduction-SVD-II.qmd

```diff
@@ -1,857 +1,759 @@
-:::
+4. Repeat step 3 until you have accounted for all $d$ directions.
 
-Next, recall the sample variance of a dataset is:
-    
-$$ \operatorname{Var}(X) = \frac{1}{m}\sum_{j=1}^{m}\Vert \mathbf{x}_j^T - \bar{\mathbf{x}}^T\Vert^2 $$
+5. Project the data into the direction of the PCs.
 
-where $\mathbf{x}_j^T$ is row $j$ of $X$.
+## Least Squares Interpretation
 
-:::: {.fragment}
-The sample variance of the set of points is the average squared distance from each point to the centroid. 
+Centered data often clusters along a line (or other low-dimensional subspace of $\mathbb{R}^{n}$).
+
+:::: {.columns}
+::: {.column width="50%"}
+![](figs/PCA_variance.png)
+The sum of **variances** (squared distances to the mean) of the projected points is a **maximum**.
+:::
+::: {.column width="50%"}
+![](figs/PCA_residual.png)
+The sum of **residuals** (squared distances from the points to the line) is a **minimum**.
+:::
 ::::
 
 :::: {.fragment}
-If we move the points (translate each point by some constant amount), the sample variance does not change.
+What is the statistical entity that measures the variability in the data?
 ::::
 
-::: {.content-visible when-profile="slides"}
-## Sample variance continued
-:::
+## Covariance Matrix
 
-So, let's move the points to be __centered__ on the __origin.__
+Let $X\in\mathbb{R}^{m\times n}$ contained the centered data. Recall that the sample covariance matrix is defined by
 
-$$ \tilde{X} = X - \mathbf{1}\overline{\mathbf{x}}^T $$
-
-:::: {.fragment}
-The sample variance of the new points $\tilde{X}$ is the same as the old points $X$, but the centroid of the new point set is the origin.
-::::
+$$
+S = \frac{1}{m-1}X^{T}X
+$$
 
-:::: {.fragment}
-Now that the mean of the points is the zero vector, we can reason geometrically.
-::::
+Example:
+Example:     
+$$
+X = \begin{bmatrix}
+5 & 0 & -5\\
+-5 & 0 & 5 \end{bmatrix} 
+$$
 
-::: {.content-visible when-profile="slides"}
-## Centered data
-:::
+$$ 
+S =  
+\begin{bmatrix}
+5 & -5\\
+0 & 0\\
+-5 & 5 \end{bmatrix} 
+\begin{bmatrix}
+5 & 0 & -5\\
+-5 & 0 & 5 \end{bmatrix}
+=
+\begin{bmatrix}
+50 & 0 & -50\\
+0 & 0 & 0\\
+-50 & 0 & 50 \end{bmatrix}
+$$
 
-Here is a picture to show why the distance-minimizing subspace is variance-maximizing.
+## Spectral Decomposition
 
-In this figure,   
+Observe that the matrix $S$ is symmetric, i.e., $S = S^{T}.$
 
-:::: {.columns}
-::: {.column width="40%"}
-* the red point is one example point from $\tilde{X}$, 
-* the green point is the origin / centroid, and
-* the blue point is the $k$-dimensional projection of the red point.
-:::
-::: {.column width="60%"}
-![](figs/pca_figure5.jpeg){fig-align="center" width="75%"}
-    
-[Source](https://liorpachter.wordpress.com/2014/05/26/what-is-principal-component-analysis/)
-:::
-::::
+Every real symmetric matrix $S$ has the factorization $V\Lambda V^{T}$, where $\Lambda$ is a diagonal matrix that contains the eigenvalues of S and teh columns of $V$ are orthogonal eigenvectors of $S$.
```

Comment:
I wanted to use V here because it is more useful in highlighting the link to SVD. I picked X in the linear algebra refresher because it is commonly used in linear algebra textbooks. However, since we change to data matrices X, it might be something to re-think.

Are you ok with keeping it as is for now?

Reply:
Fixed the typo.

Reply:
Yes. That is fine.